site/helm/installing.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Installing Weave Flux using Helm
+menu_order: 20
+---
+
+# Get started with Flux using Helm
+
+If you are using Helm already, this guide is for you. By the end
+you will have Helm installing Flux in the cluster and deploying
+any code changes for you.
+
+## Prerequisites
+
+You will need to have Kubernetes set up. To get up and running fast,
+you might want to use `minikube` or `kubeadm`. Any other Kubernetes
+setup will work as well though.
+
+*Note:* If you are using `minikube`, be sure to start the
+cluster with `--bootstrapper=kubeadm` so you are using RBAC.
+
+Download Helm:
+
+- On MacOS:
+
+  ```sh
+  brew install kubernetes-helm
+  ```
+
+- On Linux:
+  - Download the [latest release](https://github.com/kubernetes/helm/releases/latest), unpack the
+    tarball and put the binary in your `$PATH`.
+
+Now create a service account and a cluster role binding for Tiller:
+
+```sh
+kubectl -n kube-system create sa tiller
+
+kubectl create clusterrolebinding tiller-cluster-rule \
+    --clusterrole=cluster-admin \
+    --serviceaccount=kube-system:tiller
+```
+
+Deploy Tiller in `kube-system` namespace:
+
+```sh
+helm init --skip-refresh --upgrade --service-account tiller
+```
+
+## Install Weave Flux
+
+Add the Flux repository of Weaveworks:
+
+```sh
+helm repo add weaveworks https://weaveworks.github.io/flux
+```
+
+In this next step you install Weave Flux using `helm`. All you need
+to do is specify your fork of
+[flux-helm-test](https://github.com/weaveworks/flux-helm-test).
+
+ 1. Fork [flux-helm-test](https://github.com/weaveworks/flux-helm-test)
+    on Github and
+ 1. install Weave Flux and its Helm Operator by specifying your fork
+    URL.
+
+*Just make sure you replace `YOURUSER` with your GitHub username in
+the command below:*
+
+```sh
+helm install --name flux \
+--set helmOperator.create=true \
+--set git.url=git@github.com:YOURUSER/flux-helm-test \
```

Comment:
Does the chart use the same git URL for the helm operator and for the flux daemon? Hm. OK, something to fix later.

Reply:
Sounds good to do it in a later step.